Min-Ho-Lim/Portfolio
# feat: created the icon bar with github, linkedin, mail icon and mobile friendly as well

# Summary
This pull request contains the new feature; the icon bar which has Github, LinkedIn, mail icons.
Each icon has a different link to be opened.
Also, if the width of the user screen is less than 500px, the icon bar will be shown on the bottom instead of left side.

# Test
```
git fetch upstream
git checkout feat/3/create-icon-bar
npm run dev
```
1. Make sure you can see the icon bar
2. Make sure you can see a tooltip if you hover your mouse on any of three icons 
3. Make sure clicking the icon will redirect to the proper link
4. Make sure the page is responsive

# Screenshots
## Desktop
![image](https://user-images.githubusercontent.com/13793276/201496177-6a19ec0e-bb45-473b-958d-b5e710450d30.png)

## Mobile
![image](https://user-images.githubusercontent.com/13793276/201496188-bb36d8f1-3731-4d18-83fc-4dc0ae6e8a13.png)
